run.go

```diff
@@ -225,11 +227,29 @@ func goTest(args []string, stdout, stderr io.Writer, junitDir string) *testRepor
 		err: err,
 	}
 	if junitDir != "" {
-		report, err := parser.Parse(outBuf, "")
+		report, err := parser.NewParser().Parse(outBuf)
 		ret.report = &junitReport{
 			report: report,
 			err:    err,
 		}
 	}
 	return ret
 }
+
+func writeJUnitReportXML(w io.Writer, report gtr.Report) error {
```

Comment:
Copied and pasted from https://github.com/jstemmer/go-junit-report/blob/v2.0.0/internal/gojunitreport/go-junit-report.go#L73-L93.

If the master branch of go-junit-report is released, https://github.com/jstemmer/go-junit-report/blob/master/junit/junit.go#L41 can be used, but it has not been but it has not been released for a while. ref: https://github.com/jstemmer/go-junit-report/issues/164